deploy.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Why not configure the AWS environment in travis.yml?

Reply:
> Why not configure the AWS environment in travis.yml?

Because I'm not sure how this can be done.
Feel free to suggest an alternative to how things are done today

Reply:
It was shown how to do it here: https://github.com/Unee-T-INS/lambda2sqs/blob/master/.travis.yml